### PR TITLE
Fix Postprocessing in VR

### PR DESCRIPTION
**Pull request by phi-lira**
- Updated copy shader to use i.texcoordStereo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Fixed
+- Post-processing wasn't working on VR
```
Comment (Chman) [inline CHANGELOG.md]: This line is misleading, this PR fixes the first blit in stereo mode but doesn't fix the other VR issues.

**Review by Chman — CHANGES_REQUESTED** [review]
Merge conflict + changelog fix, otherwise it's good

**Comment by nerites** [comment]
Added Dennis to this PR and its corresponding PostProcessing PR, since he's doing QA on the changes.

**Comment by DennisDeRykeUnity** [comment]
Today I tested the DoubleBlitFix branch with Unity 2018.2.0a3 on Windows with Rift using the LWRP+VR template project.  I enabled each effect individually, disabling the others, and verified whether or not each effect made some reasonable noticeable change in the Rift HMD using the Oculus SDK.  In some cases I changed the parameters to extreme values to make the effects more blatant.  

I have never been able to discern whether Motion Blur does anything in VR.  I think it should lessen the jarring feeling when frames drop due to quick head movement, but maybe Time Warp in the Oculus SDK already handles that.  

Since Lens Distortions are disabled in VR mode, I verified that they work in non-VR mode.  Screen Space Reflection do not yet work with SRP, per warning message in the Inspector window, so I did not test this.

Using Depth of Field if I choose Small then Game view is very blurry, but if I choose Very Large there is very little blur.  This seems backward, but I see the same behaviour with the released SRP and Post-Proc packages, so this does not block the PR.  I might file a bug later against Unity 2018.0a3.  

I have not been able to prove to myself that Ambient Occlusion is doing anything, either in the DoubleBlitFix branch or in the released Unity 2018.3.0a3 packages, even though I set Intensity and Thickness Modifier to max values and made Color red.  My next step is to try this using QA's VR Test Suite, which includes an ambient occlusion test scene.  For now though, since I see parity with the PR branch and the released packages, this does not block the PR.    

Basically testing on Windows with Rift looks good for the PR.  For trunk PRs in the mainstream Unity codebase I would also test on a mobile XR platform prior to approving.  However, I won’t be available for more tesitng for several days.  Also, I suspect SRP/PostProc are already broken on mobile XR given Fogbugz cases such as 879280 and 1014416.  Weighing these existing mobile issues against the Windows issue that this PR fixes, plus considering the Preview nature of this package, I approve this PR.

**Review by DennisDeRykeUnity — APPROVED** [review]

**Comment by Chman** [comment]
A few notes @DennisDeRykeUnity :

> I have never been able to discern whether Motion Blur does anything in VR.

Motion blur is disabled in VR.

> I have not been able to prove to myself that Ambient Occlusion is doing anything

There's no AO support in LWRP at the moment, so it doesn't do anything.

**Comment by phi-lira** [comment]
Ok. I've updated changelog and merged it.

**Review by Chman — APPROVED** [review]

**Review by nerites — APPROVED** [review]
Again it'd be nice if the CHANGELOG described the specific issue (doubled textures per eye) that was being fixed but that's not blocking.
